common/templates/includes/cookies.njk

```diff
@@ -1 +1,5 @@
-<p>GOV.UK uses cookies to make the site simpler. <a href="#">Find out more about cookies</a></p>
+<div id="global-cookie-message" class="cookie-banner {% if seen_cookie_message %}cookie-banner--hide{% endif %}">
+  <div class="govuk-width-container">
+    <p class="govuk-body-s">GOV.UK Pay uses cookies to make the site simpler. <a class="govuk-link"href="https://www.payments.service.gov.uk/cookies/">Find out more about cookies</a></p>
```

Comment:
Any particular reason to link to Pay, rather than GOV.UK here?

Reply:
I wonder if we even need this cookie banner in this, since it is suggesting things that could be legally iffy regarding GDPR. Might be able to leave that up to the reader?